src/js/themes/grommet.js

```diff
@@ -10,6 +10,8 @@ export const grommet = deepFreeze({
     font: {
       family:
         '-apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen, Ubuntu, Cantarell, "Fira Sans", "Droid Sans",  "Helvetica Neue", Arial, sans-serif,  "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol"',
+      size: '18px',
```

Comment:
This seems too removed from how `generate()` works in base.js. For example, if `baseSpacing` is changed in base.js, the font size and line height here will not be aligned.